tests/prop-types-default-import.jsx

```diff
@@ -0,0 +1,15 @@
+import {Component} from 'react';
+import PropTypes from 'prop-types';
```

Comment:
`prop-types` does not have default import.
This should always be `import * as PropTypes...`

Reply:
So lets keep this default thingy. It is clearly wrong and in the created `d.ts` file it should be `import * as PropTypes from' instead.
The jsx file should be parsed and accepted.

Reply:
Ah, wrong. In the `d.ts` file its not required. 😄 
So, then its fine. Thanks